<launch>

  <rosparam command="load" file="$(find so_dso)/cams/kitti/00_02/T_stereo.yaml" ns="so_dso"/>

  <node name="so_dso" pkg="direct_stereo_slam" type="so_dso_node" output="screen">
    <param name="cam0_topic" value="/left/image_raw" />
    <param name="cam1_topic" value="/right/image_raw" />
    <param name="calib0" value="$(find so_dso)/cams/kitti/00_02/camera0.txt" />
    <param name="calib1" value="$(find so_dso)/cams/kitti/00_02/camera1.txt" />
    <param name="init_scale" value="20.0" />
    <param name="mode" value="1" />
    <param name="preset" value="2" />
  </node>

  </launch>
